runtime/oti/jvmimage_api.h

```diff
@@ -31,6 +31,15 @@
 extern "C" {
 #endif
 
+/*
+* Allocate memory in heap image
+* 
+* @param size[in] size to allocate
+*
+* returns pointer to allocated memory on success, NULL on failure 
+*/
+void* img_mem_allocate_mem(uintptr_t size);
```

Comment:
byteAmount

Reply:
Call this mem_allocate_memory and mem_free_memory (suggestion from [https://github.com/TobiAjila/openj9/pull/2]) Not too important